MapboxCoreNavigation/Constants.swift

```diff
@@ -147,9 +147,9 @@ let RouteControllerLinkedInstructionBufferMultiplier: Double = 1.2
 let milesToMeters = 1609.34
 
 /**
- The mimimum speed value before the user is snapped to the route. This is used to overcome inaccurate course values when a user's speed is low.
+ The minimum speed value before the user's actual location can be considered over the snapped location.
  */
-public var RouteControllerMinimumSpeedThresholdForSnappingUserToRoute: CLLocationSpeed = 2
+public var RouteControllerMinimumSpeedThresholdForUsersRealLocationInseatOfSnapped: CLLocationSpeed = 3
```

Comment:
This constant name is getting long even by [Objective-C standards](https://github.com/Quotation/LongestCocoa). How about `RouteControllerMaximumSpeedForLocationSnapping`?